src/components/Control/Popover/Popover.styles.js

```diff
@@ -5,8 +5,8 @@ export const Popover = styled(Tippy)`
   border-radius: 10px !important;
   background-color: white !important;
   box-shadow: 0 0px 16px 0 rgba(0, 0, 0, 0.2) !important;
-  width: ${({ size = 'small' }) =>
-    size === 'small' ? '170px' : '236px'} !important;
+  width: ${({ _size = 'small' }) =>
+    _size === 'small' ? '170px' : '236px'} !important;
```

Comment:
```suggestion
  width: ${({ $size = 'small' }) =>
    $size === 'small' ? '170px' : '236px'} !important;
```
props가 attr로 넘어가는 걸 방지하기 위해서 styled-components에서 권장하는 prefix로 변경하면 좋을 것 같습니다!
https://styled-components.com/docs/api#transient-props